hubblestack/extmods/grains/cloud_details.py

```diff
@@ -178,34 +205,49 @@ def _get_gcp_details():
     return ret
 
 
-def _build_gpc_extra(gcp_header, proxies):
+def _build_gcp_extra(gcp_header, proxies):
     """ Helper function to build the gcp extra dict """
-    gcp_extra = {
-        'cloud_project_id': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/project/project-id',
-            headers=gcp_header, timeout=3, proxies=proxies).text,
-        'cloud_name': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/instance/name',
-            headers=gcp_header, timeout=3, proxies=proxies).text,
-        'cloud_hostname': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/instance/hostname',
-            headers=gcp_header, timeout=3, proxies=proxies).text,
-        'cloud_zone': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/instance/zone',
-            headers=gcp_header, timeout=3, proxies=proxies).text,
-        'cloud_image': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/instance/image',
-            headers=gcp_header, timeout=3, proxies=proxies).text,
-        'cloud_machine_type': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/instance/machine-type',
-            headers=gcp_header, timeout=3, proxies=proxies).text,
-        'cloud_tags': requests.get(
-            'http://metadata.google.internal/computeMetadata/v1/instance/tags?recursive=true',
-            headers=gcp_header, timeout=3, proxies=proxies).json()}
-    interface_list = requests.get(
-        'http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces'
-        '/?recursive=true', headers=gcp_header, timeout=3, proxies=proxies).json()
+    response = requests.get('http://metadata.google.internal/computeMetadata/v1/project/project-id',
+            headers=gcp_header, timeout=3, proxies=proxies)
+    cloud_project_id = response.text if response.status_code == requests.codes.ok else None
```

Comment:
This is inconsistent with aws/azure impl.
There we raised exception for non-200 response. Here its None.